spec/features/sales_log_spec.rb

```diff
@@ -137,4 +137,72 @@
+      expect(duplicate_log.status).to eq("deleted")
+      expect(page).to have_css(".govuk-notification-banner.govuk-notification-banner--success")
+      expect(page).to have_content("Log #{duplicate_log.id} has been deleted")
+      expect(page).to have_current_path("/sales-logs/#{sales_log.id}/duplicate-logs?original_log_id=#{sales_log.id}")
+      expect(page).not_to have_content("These logs are duplicates")
+      expect(page).not_to have_link("Keep this log and delete duplicates")
+      expect(page).to have_link("Back to Log #{sales_log.id}", href: "/sales-logs/#{sales_log.id}")
+    end
+
+    it "allows changing answer on remaining original log" do
+      click_link("Keep this log and delete duplicates", href: "/sales-logs/#{sales_log.id}/delete-duplicates?original_log_id=#{sales_log.id}")
+      click_button "Delete this log"
+      click_link("Change", href: "/sales-logs/#{sales_log.id}/purchaser-code?original_log_id=#{sales_log.id}&referrer=interruption_screen")
+      click_button("Save and continue")
+      expect(page).to have_current_path("/sales-logs/#{sales_log.id}/duplicate-logs?original_log_id=#{sales_log.id}")
+      expect(page).to have_link("Back to Log #{sales_log.id}", href: "/sales-logs/#{sales_log.id}")
+    end
+
+    it "allows keeping the duplicate log and deleting the original one" do
+      expect(page).to have_current_path("/sales-logs/#{sales_log.id}/duplicate-logs")
+      click_link("Keep this log and delete duplicates", href: "/sales-logs/#{duplicate_log.id}/delete-duplicates?original_log_id=#{sales_log.id}")
+      expect(page).to have_current_path("/sales-logs/#{duplicate_log.id}/delete-duplicates?original_log_id=#{sales_log.id}")
+      click_button "Delete this log"
+      sales_log.reload
+      expect(sales_log.status).to eq("deleted")
+      expect(page).to have_css(".govuk-notification-banner.govuk-notification-banner--success")
+      expect(page).to have_content("Log #{sales_log.id} has been deleted")
+      expect(page).to have_current_path("/sales-logs/#{duplicate_log.id}/duplicate-logs?original_log_id=#{sales_log.id}")
+      expect(page).not_to have_content("These logs are duplicates")
+      expect(page).not_to have_link("Keep this log and delete duplicates")
+      expect(page).to have_link("Back to sales logs", href: "/sales-logs")
+    end
+
+    it "allows changing answers on remaining duplicate log" do
+      click_link("Keep this log and delete duplicates", href: "/sales-logs/#{duplicate_log.id}/delete-duplicates?original_log_id=#{sales_log.id}")
+      click_button "Delete this log"
+      click_link("Change", href: "/sales-logs/#{duplicate_log.id}/purchaser-code?original_log_id=#{sales_log.id}&referrer=interruption_screen")
+      click_button("Save and continue")
+      expect(page).to have_current_path("/sales-logs/#{duplicate_log.id}/duplicate-logs?original_log_id=#{sales_log.id}")
+      expect(page).to have_link("Back to sales logs", href: "/sales-logs")
```

Comment:
I'm surprised that if we have edited a value on another log, and not deleted anything, that we wouldn't want to return back into the flow of continuing to work on the log we were originally on

Reply:
That's a separate ticket. This only account for editing after duplicates have been deleted and we only have 1 log left